docs/alternator/alternator.md

```diff
@@ -3,55 +3,60 @@
 ## Introduction
 Alternator is a Scylla feature adding compatibility with Amazon DynamoDB(TM).
 DynamoDB's API uses JSON-encoded requests and responses which are sent over
-an HTTP or HTTPS transport. It is described in detail on Amazon's site:
-  https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/
+an HTTP or HTTPS transport. It is described in detail in Amazon's [DynamoDB
+API Reference](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/).
 
 Our goal is that any application written to use Amazon DynamoDB could
 be run, unmodified, against Scylla with Alternator enabled. Alternator's
 compatibility with DynamoDB is fairly complete, but users should be aware
-of some differences and some of unimplemented features. The extent of
+of some differences and some unimplemented features. The extent of
 Alternator's compatibility with DynamoDB is described in the
 [Scylla Alternator for DynamoDB users](compatibility.md) document,
-which we updated as the work on Alternator progresses and compatibility
+which is updated as the work on Alternator progresses and compatibility
 continues to improve.
 
 ## Running Alternator
 By default, Scylla does not listen for DynamoDB API requests. To enable
-such requests, you must set the `alternator-port` configuration option
-(via command line or YAML) to the port on which you wish to listen for
-DynamoDB API requests.
-
-For example., "`--alternator-port=8000`" on the command line will run
-Alternator on port 8000.
-
-By default, Scylla listens on this port on all network interfaces.
-To listen only on a specific interface, pass also an "`alternator-address`"
-option.
-
-In addition to (or instead of) serving HTTP requests on `alternator-port`,
+this API in Scylla you must set at least two configuration options,
+**alternator_port** and **alternator_write_isolation**. For example in the
+YAML configuration file:
+```yaml
+alternator_port: 8000
+alternator_write_isolation: always
```

Comment:
`always` might discourage people when they benchmark alternator out of the box, since they will pay the LWT price even if they don't need it... it was decided that this option should not have a default value, but I think that this document will de facto serve as the default - people will probably be looking for the fastest way to boot alternator and play with it, so they'll just copy-paste whatever we post here, and then forget that this write isolation option ever existed.

So perhaps we should reconsider putting  `only_rmw_uses_lwt` here? Along with a short description what it means and a note that if you're looking for stronger guarantees in a workload which mixes lwt and non-lwt operations on the same table, then you should use `always`.

Reply:
This discussion is exactly why in the previous version I didn't want to give any example of the write isolation mode :-(

But if I understood @gcarmin the user who tried it wanted such an example and not to be pointed to some other section on how to choose one, which is why I added it now.

I can indeed do what you propose and give only_rmw_uses_lwt as an example, but you're right, it means I'll need to add a short description and a warning inline, unlike what I did today (basically, if you know what this example means - go to that other section).

Reply:
Or an alternative would be to write something like that:
`alternator_write_isolation: <always|only_rmw_uses_lwt|forbid_rmw|unsafe_rmw>`